src/message.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+const GET_RANDOM_MSG_REQUEST = 'GET_RANDOM_MSG_REQUEST';
+const GET_RANDOM_MSG_SUCCSESS = 'GET_RANDOM_MSG_SUCCSESS';
+
+const getRandomMsgSuccess = (json) => {
+  return {
+    type: GET_RANDOM_MSG_SUCCSESS,
+    json,
+  };
+};
+
+//thunk
+const getRandomMsg = () => (dispacth) => {
+  dispacth({ type: GET_RANDOM_MSG_REQUEST });
+  return fetch('http://[::1]:4000/api/v1/randomMessage.json')
+    .then((response) => response.json())
+    .then((json) => dispacth(getRandomMsgSuccess(json)))
+    .catch((error) => console.log(error));
+};
```

Comment:
- [ ] Kindly remove the console log statement as suggested. It is ugly and exposes your data.